lib/envars.ml

```diff
@@ -128,7 +125,7 @@ let coqpath =
 
 (** {2 Caml paths} *)
 
-let ocamlfind () = getenv_else "OCAMLFIND" (fun () -> Coq_config.ocamlfind)
+let ocamlfind = Boot.Env.ocamlfind
```

Comment:
Maybe remove the alias? Note that this is only used twice, and reveals inconsistent usage. IMHO we should reflect access to the OCaml compiler via a handle (currently done in native and extraction)